ember/app/ui/cases/detail/index/controller.js

```diff
@@ -228,10 +228,12 @@ export default class CasesDetailIndexController extends Controller {
       }
 
       const blob = yield response.blob();
-      const filename = `${this.intl.t("documents.accountingExportFilename", {
```

Comment:
We can also get rid of this translation, right?

Reply:
Removed